contrib/cf-remote/cf_remote/remote.py

```diff
@@ -97,6 +97,48 @@ def get_info(host, *, users=None, connection=None):
         data["uname"] = ssh_cmd(connection, "uname")
         data["arch"] = ssh_cmd(connection, "uname -m")
         data["os_release"] = os_release(ssh_cmd(connection, "cat /etc/os-release"))
+
+        tags = []
+        if data["os_release"]:
+            distro = data["os_release"]["ID"]
+            major = data["os_release"]["VERSION_ID"].split(".")[0]
+            platform_tag = distro + major
+
+            # Add tags with version number first, to filter by them first:
+            tags.append(platform_tag) # Example: ubuntu16
+            if distro == "centos":
+                tags.append("el" + major)
+
+            # Then add more generic tags (lower priority):
+            tags.append(distro) # Example: ubuntu
+            if distro == "centos":
+                tags.append("rhel")
+                tags.append("el")
+        else:
+            redhat_release = ssh_cmd(connection, "cat /etc/redhat-release")
+            if redhat_release:
+                # Examples:
+                # CentOS release 6.10 (Final)
+                # Red Hat Enterprise Linux release 8.0 (Ootpa)
+                before, after = redhat_release.split(" release ")
+                distro = "rhel"
+                if before.lower().startswith("centos"):
+                    distro = "centos"
+                major = after.split(".")[0]
+                tags.append(distro + major)
+                if distro != "el":
+                    tags.append("el" + major)
+                if distro != "rhel":
```

Comment:
And here it can only be `"centos"` if it is not `"rhel"`.

Reply:
Yes, I guess my thought here was more like: don't add `"rhel"` twice to the list. I tried looking for a unique list / ordered set in python, but it seems it's not part of the standard library.

Reply:
Fixed / tried to make code more clear.